File: app/src/main/java/com/mai/nix/maiapp/navigation_fragments/ExamScheduleFragment.java
package com.mai.nix.maiapp.navigation_fragments;

import androidx.viewpager.widget.ViewPager;
import com.mai.nix.maiapp.ExamItemChooseGroupFragment;
import com.mai.nix.maiapp.ExamItemFragment;
import com.mai.nix.maiapp.TabsFragment;
import com.mai.nix.maiapp.ViewPagerAdapter;

/**
 * Created by Nix on 14.08.2017.
 */

public class ExamScheduleFragment extends TabsFragment {
    @Override
    protected void setupViewPager(ViewPager viewPager) {
        ViewPagerAdapter adapter = new ViewPagerAdapter(getChildFragmentManager());
        adapter.addFragment(new ExamItemFragment(), "Моё расписание");
        adapter.addFragment(new ExamItemChooseGroupFragment(), "Выбрать группу");
        viewPager.setAdapter(adapter);
    }
}

File: app/src/main/java/com/mai/nix/maiapp/navigation_fragments/ExamScheduleFragment.kt
package com.mai.nix.maiapp.navigation_fragments

import androidx.viewpager2.widget.ViewPager2
import com.mai.nix.maiapp.ExamItemChooseGroupFragment
import com.mai.nix.maiapp.ExamItemFragment
import com.mai.nix.maiapp.TabsFragment
import com.mai.nix.maiapp.ViewPagerAdapter

/**
 * Created by Nix on 14.08.2017.
 */
class ExamScheduleFragment : TabsFragment() {
    override fun setupViewPager(viewPager: ViewPager2) {
        val adapter = ViewPagerAdapter(requireActivity())
        adapter.addFragment(ExamItemFragment(), "Моё расписание")
        adapter.addFragment(ExamItemChooseGroupFragment(), "Выбрать группу")
        viewPager.adapter = adapter
    }
}